CHANGELOG.md

```diff
@@ -1,3 +1,268 @@
+
+These changes will also help us in our upcoming challenges to refine expand the notion of identity inside of Qri.
+
+## Step-Based Transform Execution
+We've changed the underlying way that transforms execute, dividing the work into steps, like `Setup`, `Download`, and `Transform`. With this change, we are able to report the status of their transform back to the user. It will also give the user a better sense of what is happening in their transform: how long did the `Download` step take, which step failed, which steps worked. It also paves the way for a better transform development environment in the future. Users will be able to re-run certain steps or cache steps that they don't want to waste time re-running.
+
+## Other features:
+### Change report over API
+We've added an endpoint to get the change report via the api: `/changes`. This lays the ground work for powering any future UI we have surrounding change reports in our frontend applications.
+
+### Access and Oauth
+In preparation for multi tenency, we are adding support for generating JSON web tokens that will help ensure identity on the network. You can generate an access token via the cmd using the `qri access` command, or over the api using the `/access/token` endpoint.
+
+### Stats Component!
+We've added a new component: Stats! Stats is a component that contains statistical metadata about the body of a dataset. Its purpose is to provide an "at a glance" summary of a dataset. In order to remain fast for very large dataset sizes, we have opted to calculate the stats using probabilistic structures. Stats are an important part of change reports, and allow you to get a sense of what is different in a dataset body without having to examine that body line by line. 
+
+### Apply Command
+You can now use the `qri apply` command to "dry run" and de-bug transforms! Use the `--file` flag to apply a new `transform.star` file, use a dataset reference to re-run an already existing transform on an already existing dataset, or use both to apply a new tranform to an already existing dataset. The resulting dataset is output to the terminal. To run the transform and save the results as a new commit, use the `--apply` flag in the `qri save` command.
+
+### Progress bars
+Qri has been fine tuned to handle larger datasets faster. Regardless of how fast your dataset is saving, we want to be able to track any progress. Now, when saving a new dataset or a new version of a dataset, the command line will show you a progress bar.
+
+### `qri version`
+To help with debugging, the `qri version` command now comes chock full of details, including the qri version, the build time, and the git summary.
```

Comment:
```suggestion
### `qri version`
To help with debugging, the `qri version` command now comes chock full of details, including the qri version, the build time, and the git summary. (developers: you need to use `make build` now instead of `go build`!)
```